molecule/builder-focal/dh-virtualenv.pref

```diff
@@ -0,0 +1,11 @@
+Package: *
+Pin: release a=focal
+Pin-Priority: 500
+
+Package: dh-virtualenv libjs-sphinxdoc sphinx-rtd-theme-common
```

Comment:
Note that without explicitly pinning libjs-sphinxdoc and sphinx-rtd-theme-common, installation will fail.